utils/releaser/src/ReleaseWorker/AbstractCheckPackagesTravisBuildsReleaseWorker.php

```diff
@@ -40,12 +40,20 @@ public function getDescription(Version $version): string
         return 'Check Travis build status for all packages';
     }
 
+    /**
+     * @return string
+     */
+    abstract protected function getCheckingBranchName(): string;
```

Comment:
What does the 'checkingBranch' stands for?

Reply:
It is name for branch you want to have checked. Maybe form the method name it is not really clear but I did not came up with better one.

Reply:
Maybe just `getBranchName` or `getCurrentlyCheckedBranchName`